helpers/installcredprovider.sh

```diff
@@ -6,9 +6,9 @@
 
 REPO="Microsoft/artifacts-credprovider"
```

Comment:
Please also add the instructions on how run the script to get a specific version to the comment section above so that the script so users can see that it's supported.